_posts/2025-04-06-sm9276.md

```diff
@@ -0,0 +1,84 @@
+---
+
+layout: post
+
+# If your post title is longer or more complicated
+
+# than can be represented in the filename, uncomment the following line
+
+# and specify a custom title
+
+# title: "Sample Blog Post"
```

Comment:
your title is just the filename - could you make it more descriptive of what you did?